src/utils/document-processing.ts

```
@@ -0,0 +1,356 @@
+import { MentionableDocument } from '../types/mentionable'
+import { SmartComposerSettings } from '../settings/schema/setting.types'
```

ESLint

[error] `../settings/schema/setting.types` import should occur before import of `../types/mentionable`

(import/order)